packages/jest-test-sequencer/src/index.ts

```diff
@@ -65,6 +70,23 @@ export default class TestSequencer {
     return cache;
   }
 
+  shard(
+    tests: Array<Test>,
+    options: SortOptions = {shardCount: 1, shardIndex: 1},
```

Comment:
do we need a default? I think just skipping calling this method if `globalConfig.shard` is not set is fine. At that point it's an identity function anyways